src/schema/feeds.ts

```diff
@@ -1024,24 +1052,33 @@ const anonymousFeedResolverV1: IFieldResolver<
   applyFeedPaging,
 );
 
-const feedResolverV1: IFieldResolver<unknown, Context, ConfiguredFeedArgs> =
-  feedResolver(
-    (ctx, { unreadOnly }: ConfiguredFeedArgs, builder, alias, queryParams) =>
-      configuredFeedBuilder(
-        ctx,
-        ctx.userId,
-        unreadOnly,
-        builder,
-        alias,
-        queryParams,
-      ),
-    feedPageGenerator,
-    applyFeedPaging,
-    {
-      fetchQueryParams: async (ctx) =>
-        feedToFilters(ctx.con, ctx.userId, ctx.userId),
-    },
-  );
+const feedResolverV1: IFieldResolver<
+  unknown,
+  Context,
+  ConfiguredFeedArgs & { feedId?: string }
+> = feedResolver(
+  (
+    ctx,
+    { unreadOnly, feedId }: ConfiguredFeedArgs & { feedId?: string },
+    builder,
+    alias,
+    queryParams,
+  ) =>
+    configuredFeedBuilder(
+      ctx,
+      feedId || ctx.userId,
+      unreadOnly,
+      builder,
+      alias,
+      queryParams,
+    ),
+  feedPageGenerator,
+  applyFeedPaging,
+  {
+    fetchQueryParams: async (ctx, args) =>
+      feedToFilters(ctx.con, args.feedId || ctx.userId, ctx.userId),
```

Comment:
Nitpicking but see we do like this here and above as:
```
const feedId = opts.feedId || userId;
      const filters = await feedToFilters(ctx.con, feedId, userId);
```

Reply:
Its just different places, how would you simplify it?

Reply:
Personally just do this in both 😅
feedToFilters(ctx.con, args.feedId || ctx.userId, ctx.userId)

Reply:
Ah you mean to extract it as `feedId` variable?